code/tailer/datastripe.q

```diff
@@ -118,33 +112,40 @@ upserttopartition:{[dir;tablename;keycol;enumdata;nextp]
 savetablesoverperiod:{[dir;tablename;nextp;lasttime]
     /- function to get keycol for table from access table
     keycol:`sym^.wdb.tablekeycols tablename;
-
     /- get distinct values to partition table on
     partitionlist: ?[tablename;();();(distinct;keycol)];
-
     /- enumerate and then split by keycol
     symdir:` sv dir,.proc.procname;
     enumkeycol: .Q.en[symdir;?[tablename;enlist (<;`time;nextp);0b;()]];
     splitkeycol: {[enumkeycol;keycol;s] ?[enumkeycol;enlist (=;keycol;enlist s);0b;()]}[enumkeycol;keycol;] each partitionlist;
-
     /-upsert table to partition
-    upserttopartition[dir;tablename;keycol;;nextp] each splitkeycol where 0<count each splitkeycol; 
-
+    upserttopartition[dir;tablename;keycol;;nextp] each splitkeycol where 0<count each splitkeycol;
     /- delete data from last period
-    .[.ds.deletetablebefore;(tablename;`time;lasttime)];
-    
+    ![;();0b;`symbol$()]each tablename; 
     /- run a garbage collection (if enabled)
     .gc.run[];
     };
 
-savealltablesoverperiod:{[dir;nextp;lasttime]
+savealltablesoverperiod:{[]
     /- function takes the tailer hdb directory handle and a timestamp
     /- saves each table up to given period to their respective partitions
-    savetablesoverperiod[dir;;nextp;lasttime]each .wdb.tablelist[];
+    /- totals calculates the row count of tables in .wdb.tablelist[] 
+    totals:{count get x}each .wdb.tablelist[];
```

Comment:
`get` is for loading tables from disk, I think `value` is more correct here (functionally identical but conventionally `value` is used to evaluate a symbol/string, so it's clearer what's happening in this line if we use `value`)

```suggestion
    totals:{count value x}each .wdb.tablelist[];
```

Reply:
This has been updated to use `value`